Add limitations for add/create index (#19193)
pingcap/docs: sql-statements/sql-statement-add-index.md

The `ALTER TABLE.. ADD INDEX` statement adds an index to an existing table. This operation is online in TiDB, which means that neither reads or writes to the table are blocked by adding an index.

> **Note:**
>

> > "For TiDB Dedicated clusters with 4 vCPU specifications, it is recommended to manually disable ['tidb_ddl_enable_fast_reorg'](/system-variables#tidb_ddl_enable_fast_reorg-new-in-v630) to prevent resource limitations from affecting cluster stability when adding indexes. Once disabled, indexes will be created using transactions, reducing the impact on the cluster."

Reviewer: ```suggestion
> For TiDB Dedicated clusters with 4 vCPU specifications, it is recommended to manually disable [tidb_ddl_enable_fast_reorg](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) to prevent resource limitations from affecting cluster stability during index creation. Disabling this setting allows indexes to be created using transactions, which reduces the overall impact on the cluster.
```

> > For TiDB Dedicated clusters with 4 vCPU specifications, it is recommended to manually disable [tidb_ddl_enable_fast_reorg](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) to prevent resource limitations from affecting cluster stability during index creation. Disabling this setting allows indexes to be created using transactions, which reduces the overall impact on the cluster.

Reviewer: ```suggestion
> For TiDB Dedicated clusters with 4 vCPU specifications, it is recommended to manually disable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) to prevent resource limitations from affecting cluster stability during index creation. Disabling this setting allows indexes to be created using transactions, which reduces the overall impact on the cluster.
```

The `ALTER TABLE.. ADD INDEX` statement adds an index to an existing table. This operation is online in TiDB, which means that neither reads or writes to the table are blocked by adding an index.

> <CustomContent platform="tidb-cloud">

Reviewer 2: `tidb_ddl_enable_fast_reorg` 这个变量说明的地方也加一下描述吧！

Reviewer: 加上了，PTAL https://github.com/pingcap/docs/pull/19193/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0

         
> **Note:**
>

> > For [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters with 4 vCPU specifications, it is recommended to manually disable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) to prevent resource limitations from affecting cluster stability during index creation. Disabling this setting allows indexes to be created using transactions, which reduces the overall impact on the cluster.

Reviewer: ```suggestion
> For [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters with 4 vCPU, it is recommended to manually disable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) to prevent resource limitations from affecting cluster stability during index creation. Disabling this setting allows indexes to be created using transactions, which reduces the overall impact on the cluster.
```
